packages/origin-247-claim/src/utils/Duration.ts

```diff
@@ -58,13 +58,40 @@ export class Duration {
         }
     }
 
+    public getDurationUnit() {
+        return this.durationUnit;
+    }
+
+    public roundDateUp(dateToRound: Date): number {
```

Comment:
Please add comment why this function has so strange implementation. Reminder: first day of the month used alongisde `endOf` moved date to the end of this new month, which is not the behavior we would like to have